docs/api/security/crypto.md

```diff
@@ -0,0 +1,49 @@
+## Crypto
+
+Arm Mbed Crypto is reference implementation of the cryptography interface of
```

Comment:
Adding 'the' between 'is' and 'reference' will make the sentence flow better.

Reply:
Thanks your review.